docs/developers/performance-test-setup-for-karmada.md

```diff
@@ -299,7 +300,7 @@ The meaning of args above shows as following:
 
 Since the resources of member cluster cannot be accessed in karmada control plane, we have to turn off enable-exec-service and cluster-verification.
 
-
+> Note: When the distribution of resources is complete, the resources will be immediately deleted.
```

Comment:
A question: for example, create a deployment, complete means that deployment is created or deployment is ready.

Reply:
if there is only one deployment to be distributed, it will be deleted 10s(qps is set to 0.1) after the deployment is created.

Reply:
What I want to say here is that when the whole distribution process is finished, the resources will not be saved. After the last deployment is distributed, all resources will be deleted.

Reply:
I remember there is a `timeout` parameter. How about adding it?

Reply:
I just found a parameter named `deleteAutomanagedNamespaces` in namespace object. If it is set to false, after distributing resources, CL2 will not clean resources.